bundle/bin/rke2-uninstall.sh

```diff
@@ -7,7 +7,11 @@ if [ ! $(id -u) -eq 0 ]; then
     exit 1
 fi
 
-: "${INSTALL_RKE2_ROOT:="/usr/local"}"
+if [ -f "/etc/redhat-release" ]; then
```

Comment:
We should probably match https://github.com/rancher/rke2/blob/master/install.sh#L222
```suggestion
if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
```